src/awsexplorer/defaultRegion.ts

```diff
@@ -51,9 +50,7 @@ export async function checkExplorerForDefaultRegion(
     awsContext: AwsContext,
     awsExplorer: AwsExplorer
 ): Promise<void> {
-    const credentialReaderWriter = new DefaultCredentialsFileReaderWriter()
-
-    const profileRegion = await credentialReaderWriter.getDefaultRegion(profileName)
+    const profileRegion = awsContext.getCredentialDefaultRegion()
```

Comment:
awsContext contains an [AwsContextCredentials](https://github.com/aws/aws-toolkit-vscode/blob/awschristou/credentials/src/shared/awsContext.ts#L8) object, which then backs these calls. Would it make more sense to have a single getter for that object instead of getters for each property?